index.html

```diff
@@ -0,0 +1,571 @@
+{
+  query: Query
+}
+
+type User {
+  # Specifying `as: "eg"` transforms @example into @eg
+  name: String @eg(data: ITEM)
+}
+
+# Additional specified schema elements must have their prefixes set
+# to the new name.
+#
+# This data enum was specified as `example__Data`, but will be renamed
+# as `eg__Data`:
+enum eg__Data {
+  ITEM
+}
+
+# Name transformation must also be applied to definitions pulled in from
+# specifications.
+directive @eg(data: eg__Data) on FIELD
+
+# (...other definitions omitted...)
+```
+
+## `@core__export`
+
+Control the export of individual schema elements.
+
+```graphql
+#::[The `@core__export` directive]
+directive @core__export(isExport: Boolean = true)
+  repeatable on
+  | SCALAR
+  | OBJECT
+  | FIELD_DEFINITION
+  | ARGUMENT_DEFINITION
+  | INTERFACE
+  | UNION
+  | ENUM
+  | ENUM_VALUE
+  | INPUT_OBJECT
+  | INPUT_FIELD_DEFINITION
+```
+
+`@core__export` can occur at any type system location. Elements with `@core__export` will always be included in the export schema. Elements with `@core__export(isExport: false)` will always be excluded from the export schema.
+
+## `core__FeatureUrl`
+
+```graphql
+#::[Definition of core__FeatureUrl]
+scalar core__FeatureUrl
+  @specifiedBy(url: "https://lib.apollo.dev/core/v0.1#core__featureurl")
+```
+
+Feature URLs serve two main purposes:
+  - Directing human readers to documentation about the feature
+  - Providing tools with information about the specs in use, along with enough information to select and invoke an implementation
+  
+Feature URLs **should** be [RFC 3986 URLs](https://tools.ietf.org/html/rfc3986). When viewed, the URL **should** provide the specification of the selected version of the feature in some human-readable form; a human reader should be able to click the link and go to the correct version of the docs.
+
+Although they are not prohibited from doing so, it's assumed that processors will not load the content of feature URLs. Published specifications are not required to be machine-readable, and [this spec](.) places no requirements on the structure or syntax of the content to be found there.
+
+There are, however, requirements on the structure of the URL itself:
+
+<figure>
+  <figcaption>
+    <a name=basic-anatomy-of-a-feature-url href=#basic-anatomy-of-a-feature-url class=a-header>Basic anatomy of a feature URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://spec.example.com/a/b/c/<span>exampleFeature<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span>
+  </code>
+</figure>
+
+The final two segments of the URL's [path](https://tools.ietf.org/html/rfc3986#section-3.3) **must** contain the feature's [default prefix](#prefix) and a [version tag](#versioning). The content of the URL up to and including the prefix—but excluding the version tag and trailing `/`—is the feature's *identity*. For the above example,
```

Comment:
Should this say `name` instead of `default prefix`? The figure above and the DL below say name.